tests/components/file_attachment.test.jsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+
+import FileAttachment from 'components/file_attachment.jsx';
+
+jest.mock('utils/utils.jsx', () => {
+    const original = require.requireActual('utils/utils.jsx');
+    return {
+        ...original,
+        loadImage: jest.fn((id, callback) => callback())
+    };
```

Comment:
I think `canDownloadFiles` should have been included here. Doing so will render correct snapshot for `without compact display and without can download`.
```
        loadImage: jest.fn((id, callback) => callback()),
        canDownloadFiles: jest.fn(() => true)
```

Reply:
canDownloadFiles is from utils/file_utils and loadImage is from utils/utils.

Reply:
I see, you're correct. Thanks for clarifying!